mslice/models/slice/mantid_slice_algorithm.py

```diff
@@ -31,29 +38,29 @@ def compute_slice(self, selected_workspace, x_axis, y_axis, smoothing, norm_to_o
         y_dim = workspace.getDimension(y_dim_id)
         xbinning = x_dim.getName() + "," + str(x_axis.start) + "," + str(x_axis.end) + "," + str(n_x_bins)
         ybinning = y_dim.getName() + "," + str(y_axis.start) + "," + str(y_axis.end) + "," + str(n_y_bins)
-        thisslice = BinMD(InputWorkspace=workspace, AxisAligned="1", AlignedDim0=xbinning, AlignedDim1=ybinning)
-        # perform number of events normalization then mask cells where no data was found
+        thisslice = BinMD(InputWorkspace=workspace, AxisAligned="1", AlignedDim0=xbinning, AlignedDim1=ybinning,
+                          StoreInADS=False)
```

Comment:
@martyngigg `StoreInADS` is only in nightly at the moment I think... I know that we're targetting MSlice at the nightly builds for now, and it uses the nightly build on all the instrument computers and on isiscompute, but I'm concerned that some people might be running it on the stable releases...

I guess we can leave it as it is and see if anyone complains...?

For the future, I guess we should target the current stable version from Mantid 3.12 onwards - since we've got a rough beta release deadline of Feb 2018, when 3.12 will come out?

Reply:
@mducle True. I think this is something that we'll want to sort properly when the packaging is sorted out. When we do that we can properly specify a minimum version. When your fixes for Rebin2D etc get in we will want those too I think?

If this is the only place we are using `StoreInADS` then maybe we should take it out for now and once we have 3.12 as a minimum we can tidy those places up?

Reply:
I agree, it's better to be safe since it's a small change. It'll be easy to refactor once 3.12 is out.